src/node_file.cc

```diff
@@ -564,11 +564,6 @@ static void Stat(const FunctionCallbackInfo<Value>& args) {
   ASSERT_PATH(path)
 
   if (args[1]->IsObject()) {
-
-#if ENABLE_TTD_NODE
```

Comment:
They are no longer explicitly modifying the raw double* so everything is tracked by the JsRT TTD hooks now.